Source/Scene/ModelExperimental/ModelColorPipelineStage.js

```diff
@@ -51,7 +51,17 @@ ModelColorPipelineStage.process = function (
 
   // Pass the model's color as a uniform. Set the pass type to translucent, if needed.
   const color = model.color;
-  if (color.alpha > 0.0 && color.alpha < 1.0) {
+  const renderStateOptions = renderResources.renderStateOptions;
+  if (color.alpha === 0.0) {
```

Comment:
Looks good! please also add a unit test that it sets the render state as needed.